testutil/README.md

```diff
@@ -48,13 +48,13 @@ func TestMain(t *testing.T) {
 	})
 
 	// Create a service
-	srv1.AddService("redis", "passing", []string{"master"})
+	srv1.AddService("redis", structs.HealthPassing, []string{"master"})
```

Comment:
This should work.  I ran the example in the README and it worked as expected w/ the new imports.